node-tests/local/external-link-checker.js

```diff
@@ -9,11 +9,11 @@ const { extname } = require('path');
 const { inspect } = require('util');
 const {
   findMarkdownLinks,
-	checkExternalLink,
-	mapToLocalUrl,
-	removeTrailingApostrophe,
-	randomizeFetchDelays,
-	sleep
+  checkExternalLink,
```

Comment:
The diff noise that you see here is due to mixing spaces with tabs and mixing 2 and 4 spaces for an indentation.

If you'd like you can look at only the 2nd and 3rd commits for changes described in the PR.